com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/ClientService.java

```diff
@@ -0,0 +1,9 @@
+package com.ca.lsp.cobol.service;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+public interface ClientService {
+  // TODO:provide more descriptive name for this class
```

Comment:
@grianbrcom it's time to get rid of this TODO

Reply:
Let's discuss it. Basically it should be "SettingsService". With method name "getConfiguration".
And we use the class for reading the settings. 
But we also use it for generating URI on the fly in the Middleware. Do we need to take it into account?
So, my proposal is `SettingsService::getConfiguration`.

Reply:
for me sounds ok because the LSP server doesn't know anything about middleware, he just asked for some properties from settings and ```getConfiguration``` looks appropriate to me

Reply:
I agree, sounds good